x-pack/plugins/apm/server/routes/fallback_to_transactions.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as t from 'io-ts';
+import { getFallbackToTransactions } from '../lib/helpers/aggregated_transactions/get_fallback_to_transactions';
+import { setupRequest } from '../lib/helpers/setup_request';
+import { createApmServerRoute } from './create_apm_server_route';
+import { createApmServerRouteRepository } from './create_apm_server_route_repository';
+import { kueryRt } from './default_api_types';
+
+const fallbackToTransactionsRoute = createApmServerRoute({
+  endpoint: 'GET /api/apm/fallback_to_transactions',
+  params: t.partial({ query: kueryRt }),
+  options: { tags: ['access:apm'] },
+  handler: async (resources) => {
+    const setup = await setupRequest(resources);
+    const {
+      params: {
+        query: { kuery },
+      },
+    } = resources;
+    return {
+      fallbackToTransactions: await getFallbackToTransactions({
+        ...setup,
```

Comment:
We usually don't spread `setup`
```suggestion
        setup,
```

Reply:
Fixed. We still have a lot of instances of `...setup` being spread in calls to `getSearchAggregatedTransactions` in other areas, but we can tackle that in a follow-up.